README.md

```diff
@@ -26,7 +26,7 @@ Cachet is a beautiful and powerful open source status page system.
 ## Requirements
 
 - PHP 5.5.9+ or newer
-- Apache or Nginx server
+- Http server with PHP support (eg: Apache, Nginx, Caddy)
```

Comment:
Is it just me who hates reading `HTTP` vs `Http`? It's so much nicer as a more _wordy word_... yakno